doc/nrf/known_issues.rst

```diff
@@ -1285,6 +1285,59 @@ OCT-2154: USB audio interface does not work correctly on macOS
   The audio stream is intermittent on the headset side after connecting the gateway to a Mac computer and starting audio stream.
   This issue occurs sporadically after building the nRF5340 Audio application with the default USB port as the audio source.
 
+Controller subsystem for nRF5340 Audio
+--------------------------------------
+
+The following known issues apply to the LE Audio subsystem (NET core controller) for nRF5340 used in the nRF5340 Audio application.
+
+.. rst-class:: v2-2-0
+
+OCX-138: Conformance tests
+   Not all Bluetooth conformance tests cases pass.
+
+.. rst-class:: v2-2-0
+
+OCX-152: OCX-146: 40 ms ACL interval may cause TWS unstable
+  There may be combinations of ACL intervals and other controller settings which cause instabilities to connected/true wireless stereo setups.
+
+  **Workaround:** Use an alternative connection interval.
+
+.. rst-class:: v2-2-0
+
+OCX-155: Larger timestamps than intended
+   The timestamps/SDU (Service Data Unit) references, may occasionally be larger than intended and then duplicated in the next packet.
```

Comment:
```suggestion
OCX-155: Larger timestamps than intended
   The timestamps and Service Data Unit references (SDU refs) may occasionally be larger than intended, and then duplicated in the next packet.
```

Reply:
This has to stay the way it was. Timestamps and SDU references are the same thing.

Reply:
See my new proposal. To be pushed soon.